catalog/resource_sql_table.go

```diff
@@ -0,0 +1,259 @@
+package catalog
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/databricks/databricks-sdk-go/apierr"
+	clustersApi "github.com/databricks/databricks-sdk-go/service/clusters"
+	"github.com/databricks/terraform-provider-databricks/clusters"
+	"github.com/databricks/terraform-provider-databricks/common"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type SqlColumnInfo struct {
```

Comment:
SDK - https://github.com/databricks/databricks-sdk-go/blob/main/service/unitycatalog/model.go#L128

Reply:
We decided internally to keep this model local until the unity API has full support for tables